diff -rBNu -x build -x build-ppu -x config tiff-3.9.4/contrib/iptcutil/iptcutil.c ../ps3libraries/build/tiff-3.9.4/contrib/iptcutil/iptcutil.c
--- tiff-3.9.4/contrib/iptcutil/iptcutil.c	2010-06-08 19:50:41.000000000 +0100
+++ ../ps3libraries/build/tiff-3.9.4/contrib/iptcutil/iptcutil.c	2025-09-22 13:39:22.839379297 +0100
@@ -5,7 +5,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <memory.h>
 #include <ctype.h>
 
 #ifdef HAVE_STRINGS_H
diff -rBNu -x build -x build-ppu -x config tiff-3.9.4/libtiff/tif_dirinfo.c ../ps3libraries/build/tiff-3.9.4/libtiff/tif_dirinfo.c
--- tiff-3.9.4/libtiff/tif_dirinfo.c	2010-06-09 22:15:27.000000000 +0100
+++ ../ps3libraries/build/tiff-3.9.4/libtiff/tif_dirinfo.c	2025-09-22 13:55:37.136806556 +0100
@@ -33,6 +33,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+void *
+lfind(const void *key, const void *base, size_t *nmemb, size_t size,
+      int(*compar)(const void *, const void *));
+
 /*
  * NB: NB: THIS ARRAY IS ASSUMED TO BE SORTED BY TAG.
  *       If a tag can have both LONG and SHORT types then the LONG must be
